config/initializers/secure_headers.rb

```diff
@@ -14,4 +14,10 @@
       (Rails.env.test? ? "'unsafe-inline'" : '')
     ]
   }
+
+  # So we can send JS errors to Sentry
+  # Strip off leading <pub_key>@ and trailing /<proj_num> so we have a clean
+  # domain name
+  match = (Rails.configuration.sentry_js_dsn || '').match(%r{@(.+)\/})
```

Comment:
Yes and it felt brittle - fell over if the http was missing - this seems more robust